frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/utils.tsx

```diff
@@ -1,5 +1,6 @@
 import * as _ from 'lodash';
 import { TFunction } from 'i18next';
+import { global_palette_black_400 as globalBlack400 } from '@patternfly/react-tokens';
```

Comment:
Per @dtaylor113, we should avoid importing tokens this way. See #8777 and https://bugzilla.redhat.com/show_bug.cgi?id=1952111

Reply:
Thanks @spadgett I had no idea this was an issue

Reply:
Updated all the imports to use full paths.